release/artifacts.yaml

```diff
@@ -50,7 +50,7 @@
   type: jar-scala
 - target: //daml-lf/transaction-scalacheck:transaction-scalacheck
   type: jar
-- target: //daml-lf/lfpackage:lfpackage
+- target: //daml-lf/language:language
```

Comment:
We need to update the release notes! To avoid surprises when clients updates to the latest release.

Reply:
I discussed with @gerolf-da and @cbaatz, Currently, the changelog is for SDK users. Those should not be impacted by this internal change.